Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/nbclient/client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
HTML(anim.to_html5_video())
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mRuntimeError[0m                              Traceback (most recent call last)
[0;32m/tmp/ipykernel_2137/4267696184.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0mHTML[0m[0;34m([0m[0manim[0m[0;34m.[0m[0mto_html5_video[0m[0;34m([0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/matplotlib/animation.py[0m in [0;36mto_html5_video[0;34m(self, embed_limit)[0m
[1;32m   1346[0m                 [0;31m# We create a writer manually so that we can get the[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1347[0m                 [0;31m# appropriate size for the tag[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1348[0;31m                 [0mWriter[0m [0;34m=[0m [0mwriters[0m[0;34m[[0m[0mmpl[0m[0;34m.[0m[0mrcParams[0m[0;34m[[0m[0;34m'animation.writer'[0m[0;34m][0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1349[0m                 writer = Writer(codec='h264',
[1;32m   1350[0m                                 [0mbitrate[0m[0;34m=[0m[0mmpl[0m[0;34m.[0m[0mrcParams[0m[0;34m[[0m[0;34m'animation.bitrate'[0m[0;34m][0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/matplotlib/animation.py[0m in [0;36m__getitem__[0;34m(self, name)[0m
[1;32m    149[0m         [0;32mif[0m [0mself[0m[0;34m.[0m[0mis_available[0m[0;34m([0m[0mname[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    150[0m             [0;32mreturn[0m [0mself[0m[0;34m.[0m[0m_registered[0m[0;34m[[0m[0mname[0m[0;34m][0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 151[0;31m         [0;32mraise[0m [0mRuntimeError[0m[0;34m([0m[0;34mf"Requested MovieWriter ({name}) not available"[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    152[0m [0;34m[0m[0m
[1;32m    153[0m [0;34m[0m[0m

[0;31mRuntimeError[0m: Requested MovieWriter (ffmpeg) not available
RuntimeError: Requested MovieWriter (ffmpeg) not available

